scripts/generate-mocks.js

```diff
@@ -33,4 +47,13 @@ hasChanges(inputPaths).then((hasChanged) => {
   const subFolder = '/components/';
   // core input path is not needed to generate angular mocks
   new GenerateMocks().renderMocks(inputPaths.angular, outputPaths, subFolder);
-});
+}
+
+if (hasFlag('force')) {
+  generateMocks();
+} else {
+  hasChanges(inputPaths).then((hasChanged) => {
+    if (!hasChanged) return;
+    generateMocks();
+  });
+}
```

Comment:
```suggestion
  hasChanges(inputPaths).then((hasChanged) => {
    if (!hasChanged || !hasFlag('force')) return;
    generateMocks();
  });
```

Feel free to leave as is; personally i just feel like the above is a bit more readable :)